packages/math/src/rounding.ts

```diff
@@ -0,0 +1,87 @@
+import { Dec } from "@keplr-wallet/unit";
+
+import { BigDec } from "./big-dec";
+
+export function compareBigDec_checkMultErrorTolerance(
```

Comment:
What do you think about having it in camelCase instead of a mix of snake case and camel? 

```suggestion
export function compareBigDecCheckMultErrorTolerance(
```

Reply:
Why not make it generic to the type of decimal being used?